crates/swc_css_lints/tests/rules/fail/custom-property-no-missing-var-function/input.css

```diff
@@ -0,0 +1,7 @@
+a { color: --foo; }
+a { color: env(--foo); }
+a { color: color(--foo 0% 0% 0% 0%); }
+a { --foo: red; color: --foo; }
+@property --foo {} a { color: --foo; }
+:root { --foo: pink; }  a { color: --foo, red; }
+:root { --bar: 0; } a { color: --foo(--bar); }
```

Comment:
Can we also add test for `background-color: var(--foo, var(--bar))` (i.e. `var` in `var`) and also in other functions, for example - `width: calc(var(--variable-width) + 20px);`

Reply:
For inside other functions like `width: calc(var(--variable-width) + 20px)`, this is already included.